src/main/kotlin/net/irisshaders/lilybot/extensions/moderation/TemporaryModeration.kt

```diff
@@ -113,40 +115,83 @@ class TemporaryModeration : Extension() {
 							content = "You cannot warn bot users!"
 						}
 						return@action
-					// If the moderator ping role is in roles, return
+						// If the moderator ping role is in roles, return
 					} else if (moderatorRoleId in roles) {
 						respond {
 							content = "You cannot warn moderators!"
 						}
 						return@action
 					}
-				// Just to catch any errors in the checks
+					// Just to catch any errors in the checks
 				} catch (exception: Exception) {
 					logger.warn("IsBot and Moderator checks skipped on `Warn` due to error")
 				}
 
-				val oldPoints = DatabaseHelper.getWarn(userArg.id, guild!!.id)
-				val newPoints = oldPoints.plus(arguments.warnPoints)
-				DatabaseHelper.setWarn(userArg.id, guild!!.id, newPoints)
-				val databasePoints = DatabaseHelper.getWarn(userArg.id, guild!!.id)
+				val oldStrikes = getWarn(userArg.id, guild!!.id)
+				DatabaseHelper.setWarn(userArg.id, guild!!.id, oldStrikes.plus(1))
+				val newStrikes = getWarn(userArg.id, guild!!.id)
 
 				// DM the user about the warning
 				val dm = userDMEmbed(
 					userArg,
 					"You have been warned in ${guild?.fetchGuild()?.name}",
-					"You were given ${arguments.warnPoints} points\n" +
-							"Your total is now $databasePoints\n\n**Reason:**\n${arguments.reason}",
+					"You were given a strike\n" +
+							"Your total is now $newStrikes\n\n**Reason:**\n${arguments.reason}\n\nFor more information" +
+							" about the warn system, please see [this document](https://github.com/IrisShaders/LilyBot" +
+							"/blob/develop/docs/warn-strikes.md)",
 					null
 				)
 
-				// Check the points amount, before running sanctions
-				if (databasePoints in (75..124)) {
+				respond {
+					content = "Warned User"
```

Comment:
User shouldn't be capitalized. If we're being consistent with other commands, there should be a period after as well.